qiita_pet/handlers/analysis_handlers.py

```diff
@@ -307,13 +307,21 @@ def get(self):
         # Format sel_data to get study IDs for the processed data
         sel_data = defaultdict(dict)
         proc_data_info = {}
-        sel_samps = Analysis(self.current_user.default_analysis).samples
+        sel_samps = self.current_user.default_analysis.samples
         for pid, samps in viewitems(sel_samps):
-            proc_data = ProcessedData(pid)
+            proc_data = Artifact(pid)
             sel_data[proc_data.study][pid] = samps
             # Also get processed data info
-            proc_data_info[pid] = proc_data.processing_info
-            proc_data_info[pid]['data_type'] = proc_data.data_type()
+            # TODO plugin:
```

Comment:
Just a note - we will be able to duplicate this information with the artifacts, however, this is highly tied up with target gene and is not gonna work for other datatypes - so we should rethink this information.